plugins/Eq/.directory

```diff
@@ -0,0 +1,4 @@
+[Dolphin]
```

Comment:
The file `plugins/Eq/.directory` should be removed from this commit as it is not needed as part of the LMMS source code.